### ISSUE TITLE
MNIST example is missing batchnorm for DCGAN

### ISSUE THREAD

**Issue by marc-tonsen** [igul222/improved_wgan_training#23]
In the MNIST example the batch normalization is only added for mode 'wgan', which seems to break the example when run in mode 'dcgan'. Adding batch normalization in mode 'dcgan' too fixes it.
